opta/core/plan_displayer.py

```diff
@@ -64,62 +115,80 @@ def display(detailed_plan: bool = False) -> None:
 
             if not address.startswith("module."):
                 logger.warn(
-                    f"Unable to determine severity of changes to resource {address}. "
+                    f"Unable to determine risk of changes to resource {address}. "
                     "Please run in detailed plan mode for more info"
                 )
             module_name = address.split(".")[1]
             module_changes[module_name] = module_changes.get(
-                module_name, {"severity": BENIGN_SEVERITY, "resources": {}}
+                module_name, {"risk": LOW_RISK, "resources": {}}
             )
             resource_name = ".".join(address.split(".")[2:])
             actions = resource_change.get("change", {}).get("actions", [])
             if "create" in actions and "delete" in actions:
                 actions = ["replace"]
             action = actions[0]
-            current_severity = ACTION_TO_SEVERITY[action]
-            action_reason = resource_change.get("action_reason", "N/A")
+            if action in ["read", "create"]:
+                current_risk = LOW_RISK
+                action_reason = (
+                    "data_refresh" if action == "read" else "ground_up_creation"
+                )
+            elif action in ["replace", "destroy"]:
+                current_risk = HIGH_RISK
+                action_reason = resource_change.get("action_reason", "N/A")
+            elif action in ["update"]:
+                current_risk, action_reason = PlanDisplayer.handle_update(resource_change)
+            else:
+                raise Exception(f"Do not know how to handle planned action: {action}")
+
             module_changes[module_name]["resources"][resource_name] = {
                 "action": action,
                 "reason": action_reason,
-                "severity": current_severity,
+                "risk": current_risk,
             }
-            module_changes[module_name]["severity"] = _max_severity(
-                module_changes[module_name]["severity"], current_severity
+            module_changes[module_name]["risk"] = _max_risk(
+                module_changes[module_name]["risk"], current_risk
             )
-            plan_severity = _max_severity(plan_severity, current_severity)
+            plan_risk = _max_risk(plan_risk, current_risk)
 
         logger.info(
-            f"Identified total severity of {SEVERITY_COLORS[plan_severity]}{plan_severity}{attr(0)}.\n"
-            f"{SEVERITY_EXPLANATIONS[plan_severity]}\n"
+            f"Identified total risk of {RISK_COLORS[plan_risk]}{plan_risk}{attr(0)}.\n"
+            f"{RISK_EXPLANATIONS[plan_risk]}\n"
             "If you want extra help, please feel free to reach out to the Runx team at https://slack.opta.dev.\n"
```

Comment:
For additional help, please reach out to the RunX team at https://slack.opta.dev/